src/main/kotlin/lotto/domain/WinningResult.kt

```diff
@@ -0,0 +1,27 @@
+package lotto.domain
+
+object WinningResult {
+    private val mWinningResult: MutableMap<Rank, Int> = mutableMapOf(
+        Rank.FIFTH to 0,
+        Rank.FOURTH to 0,
+        Rank.THIRD to 0,
+        Rank.SECOND to 0,
+        Rank.FIRST to 0
+    )
+    private val winningResult: Map<Rank, Int>
+        get() = mWinningResult
```

Comment:
컬렉션이 외부로 노출되는 경우에는 방어적 복사를 해주세요.
```suggestion
        get() = mWinningResult.toMap()
```

Reply:
불변객체에 계속 주의해야겠네요!